### DISCUSSION TITLE
We need chinese documents!!!

### DISCUSSION THREAD

**Discussion by nilsir** [symfony/symfony#51125] [Q&A]
In China, there are many php developers using symfony to build applications, but until now the official documents do not support Chinese, so Chinese users are very confused, which is not conducive to the dissemination and learning of the framework, eagerly hope that the official Chinese documents can be published, thank you support

**Comment by MatTheCat** [comment]
Hello,

the official documentation will only ever be available in english; see https://symfony.com/blog/discontinuing-the-symfony-community-translations and https://symfony.com/blog/discontinuing-the-community-translations
